mk/external.mk

```diff
@@ -58,10 +59,13 @@ define verify
                 | cut -f 1 -d ' ' > $(SHA1_FILE2) && cmp $(SHA1_FILE1) $(SHA1_FILE2))), \
             ($(eval VERIFIER := echo "$(strip $(1))  $(strip $(2))" | $(SHA1SUM) -c)) \
     ))
-    $(eval _ := $(shell $(VERIFIER)))
+    $(eval _ := $(shell $(VERIFIER) 2>&1))
```

Comment:
Why make this changes?

Reply:
This redirects `stderr` to `stdout`. The output message after running `make artifact` will be in a mess if every SHA1 verifications fail without redirecting the message.

Reply:
> This redirects `stderr` to `stdout`. The output after running `make artifact` will be in a mess if every SHA1 verifications fail without redirecting error message.

Got it.